client/src/components/shared/SaveBar.tsx

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import {ClearRounded, SaveOutlined} from '@mui/icons-material';
+import {Box, Button, Fade, Typography, useTheme} from '@mui/material';
+const SaveBar = ({
```

Comment:
Add a newline between the imports and the code (should probably be an eslint rule)